src/utils/string.formatter.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * String formatting functions
+ */
+const PRINTABLE_CHARS: Set<String> =
```

Comment:
shouldn't these be chars?

Reply:
ignore me, charAt returns a string!

Reply:
Unfortunately, this aint java

Reply:
The set requires a generic type you see.

Reply:
One last tiny thing from me, then i'll approve I swear.
Sonar has produced a smell, we should use primitive string here, instead of the wrapper type String

http://code-analysis.platform.aws.chdev.org:9000/project/issues?id=uk.gov.companieshouse%3Astrike-off-objections-web&open=AXVqfUZFbKZCo3W_hI8L&pullRequest=137&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL

Reply:
Right using "string"